org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -1470,28 +1491,32 @@ protected void openReportInViewer(int AD_Process_ID, int AD_PrintFormat_ID, Stri
    		new ZkReportViewerProvider().openViewer(re);
    	}
 
-	private void fillParameter(MPInstance pInstance, String parameters) {		
+   	/**
+   	 * Fill Parameters
+   	 * @param pInstance
+   	 * @param parameters
+   	 * @return true if the parameters were filled successfully 
+   	 */
+	private boolean fillParameter(MPInstance pInstance, String parameters) {	
+		MProcessPara[] processParams = pInstance.getProcessParameters();
 		if (parameters != null && parameters.trim().length() > 0) {
-			Map<String, String> paramMap = new HashMap<String, String>();
-			String[] params = parameters.split("[,]");
-			for (String s : params)
-			{
-				int pos = s.indexOf("=");
-				String key = s.substring(0, pos);
-				String value = s.substring(pos + 1);
-				paramMap.put(key, value);
-			}
-			MProcessPara[] processParams = pInstance.getProcessParameters();
+			Map<String, String> paramMap = MDashboardContent.parseProcessParameters(parameters);
 			for (int pi = 0; pi < processParams.length; pi++)
 			{
 				MPInstancePara iPara = new MPInstancePara (pInstance, processParams[pi].getSeqNo());
 				iPara.setParameterName(processParams[pi].getColumnName());
 				iPara.setInfo(processParams[pi].getName());
 				
+				MProcessPara sPara = processParams[pi];
+				
 				String variable = paramMap.get(iPara.getParameterName());
 
-				if (Util.isEmpty(variable))
-					continue;
+				if (Util.isEmpty(variable)) {
```

Comment:
safer to use Util.isEmpty(variable, true) here

Reply:
Thank you, changes applied